### PR TITLE
Fix various rubocop/codeclimate issues

### PR DESCRIPTION
**Pull request by tchia04**
This should fix the following issue at https://codeclimate.com/github/bundler/gemstash/builds/115
```
/usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:122:in `require': cannot load such file -- rubocop-performance (LoadError)
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:122:in `require'
	from /usr/local/bundle/gems/rubocop-0.52.1/lib/rubocop/config_loader_resolver.rb:15:in `block in resolve_requires'
```

### CODE REVIEW

**Comment by tchia04** [comment]
I don't know why the codeclimate is not working here. Maybe it's disabled? 
It works fine in my fork at https://github.com/tchia04/gemstash/pull/2

**Comment by bronzdoc** [comment]
Hey @tchia04 We don't use codeclimate in this repo, so i don't really understand what is this fixing.

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
   fixme:
     enabled: true
   rubocop:
     enabled: true
-ratings:
-  paths:
-  - "**.rb"
-exclude_paths:
-- spec/
+    # Avaiable channels: https://docs.codeclimate.com/docs/rubocop
```
Comment (olleolleolle) [inline .codeclimate.yml]: Tiny typo, should be "available"

**Comment by tchia04** [comment]
I am not sure how to squash all the commit into one. I counted the number of commits from HEAD and pass it to git rebase -i HEAD~X and it turns out to either pick up too many commits or too less commits. I believe there is an option to squash the commit on the merge button.  Could you try that

**Comment by bronzdoc** [comment]
closed in favor of https://github.com/rubygems/gemstash/pull/241

**Pull request closed without merging**